src/main/kotlin/com/kotlindiscord/bot/filtering/InviteFilter.kt

```diff
@@ -0,0 +1,163 @@
+package com.kotlindiscord.bot.filtering
+
+import com.gitlab.kordlib.common.entity.ChannelType
+import com.gitlab.kordlib.common.entity.Snowflake
+import com.gitlab.kordlib.core.cache.data.InviteData
+import com.gitlab.kordlib.core.entity.Invite
+import com.gitlab.kordlib.core.entity.Message
+import com.gitlab.kordlib.core.entity.User
+import com.gitlab.kordlib.core.entity.channel.Channel
+import com.gitlab.kordlib.core.event.message.MessageCreateEvent
+import com.gitlab.kordlib.core.event.message.MessageUpdateEvent
+import com.kotlindiscord.bot.deleteIgnoringNotFound
+import com.kotlindiscord.kord.extensions.ExtensibleBot
+import mu.KotlinLogging
+
+private val logger = KotlinLogging.logger {}
+
+@Suppress("MagicNumber", "UnderscoresInNumericLiterals")  // They're guild IDs
+private val whitelist = setOf(  // Guilds we allow invites to
+    Snowflake(705370076248932402),  // Kotlin Discord
+    Snowflake(267624335836053506),  // Python Discord
+    Snowflake(280033776820813825),  // Functional Programming
+    Snowflake(273944235143593984),  // STEM
+    Snowflake(590806733924859943),  // Discord Hack Week
+    Snowflake(197038439483310086),  // Discord Testers
+    Snowflake(81384788765712384),   // Discord API
+    Snowflake(613425648685547541)   // Discord Developers
+)
```

Comment:
We probably want to make this a config entry to make it easier to edit.

Reply:
I think there's a heck of a lot more that could be configurable about this extension, honestly. Is it worth doing this now? Or is it better to move everything at once? I'm not sure.

Reply:
Yeah I guess that can wait